overrides/default-settings.gschema.override

```diff
@@ -136,18 +135,10 @@ active=false
 [org.gnome.settings-daemon.plugins.color]
 night-light-temperature=4500
 
-[org.gnome.settings-daemon.plugins.cursor]
-# Workaround upstream gnome-settings-daemon bug (https://bugzilla.gnome.org/show_bug.cgi?id=694758) as tracked by elementary (https://bugs.launchpad.net/elementaryos/+bug/1248747)
-active=false
```

Comment:
Ooh, this we can probably drop. I wonder if it has anything to do with the cursor size stuff, too?

Reply:
I've tested this and it does seem to improve the cursor being the same size, and it looks like it also fixes support for the cursor size setting.

https://github.com/elementary/gala/issues/300